gfootball/doc/compile_engine.md

```diff
@@ -0,0 +1,112 @@
+# Compiling GFootball Engine #
+
+## Windows
+Prerequisites:
+- [Git](https://git-scm.com/download/win),
+- [CMake](https://cmake.org/download/),
+- [Visual Studio 2019 Community Edition](https://visualstudio.microsoft.com/downloads/) make sure to 
```

Comment:
I was used VS 2017 to build, not only require to use VS 2019, but may be we should consider using newest IDE to keep away using boost